mms/metric.py

```diff
@@ -29,7 +30,9 @@ class Metric(object):
     """Metric class for model server
```

Comment:
Can you please add proper docstring for this ctor and all input args?

Reply:
updated.